Facebook.Unity/Results/GraphResult.cs

```diff
@@ -50,19 +50,16 @@ private void Init(string rawResult)
                 return;
             }
 
-            object serailizedResult = MiniJSON.Json.Deserialize(this.RawResult);
```

Comment:
This is the only like where we deserialize the graph result. We don't deserialize it more than once